tests/TestHierarchy.hs

```diff
@@ -0,0 +1,26 @@
+module TestHierarchy where
+
+import System.Process (readCreateProcess, shell)
+
+import Test.Hspec (Spec, describe, hspec, it, shouldBe)
+
+import TestUtils (pushd)
+
+main :: IO ()
+main = pushd "tests/support/hierarchy" (hspec spec)
+
+spec :: Spec
+spec = do
+  describe "hierarchy" $ do
+    it "generates usable graphviz graphs" $ do
+      let expected = unlines
+            [ "digraph Main {"
+            , "  A;"
+            , "  A -> B;"
+            , "  C;"
+            , "}"
+            ]
+
+      actual <- readCreateProcess (shell "purs hierarchy src/Main.purs") ""
```

Comment:
Well wait, it sort of feels like we should have more than one test. Would you be against having at least one test that ensures everything is wired up properly for the command people would use?

I don't think hierarchy is really used much, but it'd be nice if breakages didn't take years to find out.